_posts/2019-09-09-freshers-week-2019.md

```diff
@@ -0,0 +1,143 @@
+
+Location: Ming Moon Restaurant, 8 Wrottesley St, Birmingham B5 4RT
+
+Facebook Event: [link](https://www.facebook.com/events/2503094343110792/)
+
+### Pub Night
+
+This is our first big pub night of the year! We've booked the function room in the Bristol Pear just for you! Join us for a drink and get to know your coursemates. Whether you're a fresher or not, come to the Pear for a great night!
+
+Time and Date: Friday 27<sup>th</sup> September, 18:30
+
+Location: (Upstairs in the) Bristol Pear, 676 Bristol Rd, Birmingham B29 6BJ
+
+Facebook Event: [link](https://www.facebook.com/events/416995638948699/)
+
+### Welcome Talks
+
+If you've been wondering what you can do with Computer Science Society, we'll introduce you. We will be telling you what events we will be holding throughut the year and what you can expect to be doing. You can also find out about things like hackathons and more about the local tech industry in Birmingham.
+
+Time and Date: Wednesday 2<sup>nd</sup> October, 15:00
+
+Location: TBC - keep an eye on the Facebook Event and our social media.
+
+Facebook Event: [link coming soon]()
+
+### EPS Societies' Fair
+
+We will be having a stand at the EPS Societies' Fair. Come and sya hi to us and have a chat to other societies to find out more about the great things you can get involved with while studyng at the university. EPS is the College of Engineering and Physical Sciences, of which the School of Computer Science is a part of. 
+
+Time and Date: Wednesday 9<sup>th</sup> October, 9:00 - 14:00
+
+Location: Chancellor's Court (Aston Webb Semi-circle)
+
+### MLH Local Hack Day
+
+Get a mini-taster into the world of hackathons! This year, MLH (Major League Hacking) is running it slightly differently. The first of the days we are running is "Learn", during which we are going to run some workshops during which you can learn some new skills. Follow our social channels and keep an eye out for further updates. 
+
+This event is run in coperation with Aston Computer Science Society at Aston Univeristy, and Hackathon and Computing Society at Birmingham City Univeristy.
+
+Find out more [here](https://localhackday.mlh.io/learn/).
+
+Time and Date: Saturday 12<sup>th</sup> October
+
+Location: TBC, University of Birmingham
+
+### Linux InstallFest (with AFNOM)
+
+We're working with AFNOM to bring you a Linux install workshop. Linux is the preferred operating system of many developers and mnay students aren't quite familair with this yet. This workshop is designed to help you become familiar with Linux and how to start using it. 
+
+AFNOM is one of the EPS Societies, and you can find more out about them here: [link](https://afnom.net)
+
+Time and Date: Wednesday 16<sup>th</sup> October
+
+Location: TBC
+
+### Tech Talk
+
+This is our first Tech Talk of the year. Learn something new and we'll bring some free pizza for you. Follow our social channels and keep an eye out for further updates. 
```

Comment:
Maybe mention who's doing the talk?

Reply:
WIll give more details when Jon confirms